fiftyone/core/state.py

```diff
@@ -233,7 +234,9 @@ def serialize_fields(schema: t.Dict) -> t.List[SampleField]:
                     embedded_doc_type=embedded_doc_type,
                     subfield=subfield,
                     description=field.description,
-                    info=field.info,
+                    info=dict(**field.info)
+                    if isinstance(field.info, BaseDict)
+                    else field.info,
```

Comment:
cool! this doesn't seem directly related to similarity search but probably got surfaced when computing it;